invenio_files_rest/__init__.py

```diff
@@ -386,6 +387,14 @@
 Be aware that there are security implications to take into account when
 serving files. See the :ref:`usage-security` for more information.
 
+Invenio-Files-Rest provides also the functionality to serve your files directly
+from your external storage. This is achieved by attaching the
+`X-Accel-Redirect` header to the response, which will then be redirected by
+your Web Proxy (e.g. NGINX, Apache) to your external storage, finally streaming
+the file directly to the user.
+To use this you will need to configure your Web Proxy accordingly and then
```

Comment:
Comment: what do you think of:
```suggestion
To use this feature you will need to configure your Web Proxy accordingly and then
```